src/dataset-builder/ConceptSetCreator.ts

```diff
@@ -1,15 +1,16 @@
 import _ from 'lodash/fp';
 import { h } from 'react-hyperscript-helpers';
 import { DomainConceptSet } from 'src/dataset-builder/DatasetBuilderUtils';
-import { DatasetModel, SnapshotBuilderConcept as Concept } from 'src/libs/ajax/DataRepo';
+import { Snapshot, SnapshotBuilderConcept as Concept, SnapshotBuilderSettings } from 'src/libs/ajax/DataRepo';
```

Comment:
Thoughts on renaming to `SnapshotModel`

Reply:
That would be reasonable - I think I want to keep it Snapshot just because I think it is used across other parts of the terra ui.